build-node.js

```diff
@@ -2,11 +2,11 @@ import fs from 'fs';
 import path from 'path';
 import ncc from '@zeit/ncc';
 
-ncc(path.resolve(process.cwd(), 'src/system-node.js'), { filename: 'system-node.cjs' })
+ncc(path.resolve(process.cwd(), 'src/system-node.js'), { filename: 'system-node.cjs', sourceMap: false })
 .then(({ code, map, assets }) => {
   if (Object.keys(assets).length)
     console.warn('Found build assets - these may need to be included.');
   console.log("Writing dist/system-node.cjs");
   fs.writeFileSync(path.resolve(process.cwd(), 'dist/system-node.cjs'), code);
-  fs.writeFileSync(path.resolve(process.cwd(), 'dist/system-node.cjs.map'), map);
+  // fs.writeFileSync(path.resolve(process.cwd(), 'dist/system-node.cjs.map'), map);
```

Comment:
Were source maps intentionally turned off here?

Reply:
They've never been turned on - which requires writing the register file (from the output asset list) as well as setting the `sourceMap: true` option. I'm open to a PR here.

Reply:
I see 👍